### ISSUE TITLE
Question: Is it possible to compile for UE v4.5.1?

### ISSUE THREAD

**Issue by matt-d-rat** [VSZue/DonAINavigation#11]
Before I go down a rabbit hole exploring whether this is possible, I wanted to ask whether there is any reason why this plugin could not be conceivably compiled for UE4 v4.5.1?

Before you ask, I realize v4.5.1 is miles behind the current version of UE4, but unfortunately I am tied into this version as it is the version of the engine that is used by the game that I mod.

This leads me to another question: Would I encounter any issues in building this plugin for a version of UE4 which is essentially a developer kit for this game? Ie: I don't have access to some of the menus normally present in the standard UE4 editor such as making a new C++ project or C++ class from UE4 itself.

I do have Visual Studio installed, however I'm not a C++ programmer though (I work in web technologies), but can usually follow instructions competently enough when provided.

Basically what I am trying to ask is whether this would be worth my time, or am I likely to bang my head against a brick wall trying to get it to work? Failing all else, would anyone else be able to compile a version for v4.5.1 for me?

Many thanks for any help or advise offered. Keeping my fingers crossed that I can make use of this awesome plugin - it definitely would solve my use case.

**Comment by matt-d-rat** [comment]
I'm guessing the short answer is no. I took a stab at it myself and I got as far as Generating Headers (after making a few changes) before my ability to decipher the compiler error messages reached its limit. As I mentioned before, I'm not a C++ programmer, so I am pretty surprised I got this far.

If anyone has more experience and would like to take a stab at compiling this plugin for UE 4.5.1 then I would greatly appreciate it.
